.github/workflows/test.yml

```diff
@@ -44,3 +44,32 @@ jobs:
         HEADLESS: 1
       run: |
         bash make/build.sh --skip-download
+
+  freebsd-x64:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run tests in FreeBSD
+        id: test
+        uses: vmactions/freebsd-vm@v1
+        with:
+          prepare: |
+            pkg install -y bash curl git gmake gnugrep openjdk21 zip
+            mount -t fdescfs fdesc /dev/fd
+            mount -t procfs proc /proc
+
+          run: |
+            pwd
+            ls -lah
+            whoami
+            env
+            freebsd-version
+            sysctl hw.model
+            sysctl hw.ncpu
+            sysctl hw.physmem
+            sysctl hw.usermem
+            pw user add -n action -m
+            su action -c 'git config --global --add safe.directory /home/runner/work/jtreg/jtreg'
+            su action -c 'bash make/build.sh --jdk /usr/local/openjdk21'
```

Comment:
I assume that the hard-coded `/usr/local/openjdk21` is well-defined and stable for `pkg ... openjdk21 ...` calls. Or is there an environment variable (`JAVA_HOME`?) being set that can be read out and used here?

Reply:
While it's technically possible to change the prefix from `/usr/local`, this is the default unless explicitly changed.

There are ways to get the path without hardcoding it, but I'm not sure it's worth the effort here:

```
% JAVAVM_DRYRUN=1 javavm 2>/dev/null |grep JAVA_HOME|cut -d '=' -f 2
/usr/local/openjdk21
```

Reply:
The default is good-enough here, I guess. Otherwise, it would be more elegant to specify a custom location for the installation and then pass it later to subsequent processes.